app/models/katello/concerns/host_managed_extensions.rb

```diff
@@ -32,6 +33,15 @@ def rhsm_organization_label
         self.organization.label
       end
 
+      def info_with_katello
+        info = info_without_katello
+        info['parameters']['kt_env'] = self.lifecycle_environment.try(:label) #deprecated
+        info['parameters']['kt_cv'] = self.content_view.try(:label) #deprecated
```

Comment:
Do you want to formally start a deprecation cycle on these? We could include a warning here similar to what @johnpmitsch did with the distributors so that by 2.5 or 2.6 we can remove these.